packages/flutter_tools/lib/src/hot.dart

```diff
@@ -446,7 +446,22 @@ class HotRunner extends ResidentRunner {
     final bool shouldReportReloadTime = !_runningFromSnapshot;
     Stopwatch reloadTimer = new Stopwatch();
     reloadTimer.start();
+    Stopwatch devFSTimer;
```

Comment:
There's more measuring logic than the useful logic :) I admit, I can't think of a less intrusive way to measure it though, so lgtm.